src/App.js

```diff
@@ -1,24 +1,47 @@
-import logo from './logo.svg';
-import './App.css';
+import * as React from "react";
+import Product from "../src/components/Product";
+import Box from "@mui/material/Box";
+import Grid from "@mui/material/Grid";
+import axios from "axios";
 
 function App() {
+  const [products, setProducts] = React.useState([]);
+  const getAllProducts = () => {
+    axios
+      .get(`http://localhost:3000/`)
+      .then((res) => {
+        setProducts(res.data.product);
+      })
+      .catch((error) => {
+        console.log(`error`);
+      });
+  };
+  React.useEffect(() => {
+    getAllProducts();
+  }, []);
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
+    <>
```

Comment:
So a `return` statement expects a single node to return. I see you've ensured that with the fragment `<></>`, but even if we remove it, `Box` is already a single parent node.